src/components/connection_handler/test/connection_handler_impl_test.cc

```diff
@@ -413,15 +417,22 @@ TEST_F(ConnectionHandlerTest, GetAppIdOnSessionKey) {
   AddTestSession();
 
   uint32_t app_id = 0;
+  DeviceHandle test_handle;
   const uint32_t test_id = SessionHash(uid_, out_context_.new_session_id_);
 
-  connection_handler::DeviceHandle* device_id = NULL;
   EXPECT_EQ(0,
             connection_handler_->GetDataOnSessionKey(
-                connection_key_, &app_id, NULL, device_id));
+                connection_key_, &app_id, nullptr, &test_handle));
```

Comment:
@iCollin this is because we have a separate test for each param. This particular UT verifies app_id param. I removed redundant param to avoid confusions.
See https://github.com/smartdevicelink/sdl_core/pull/2682/commits/0b92f716443b0246f9af8210846e8b2c53fa7221